docs/source/gbd2019_models/risk_attributable_causes/protein_energy_malnutrition/index.rst

```diff
@@ -4,6 +4,72 @@
 Protein Energy Malnutrition
 ===========================
 
+Protein-energy malnutrition (PEM) is an acute form of undernutrition. 
+Undernutrition can take a variety of forms, including stunting (low height for 
+age), wasting (low weight for height), underweight (low weight for age), and 
+micronutrient deficiencies (a lack of key vitamins or minerals). PEM refers 
+specifically to an acute (as opposed to chronic) disorder that results in 
+wasting, and is classified as either marasmus, kwashiorkor, or some 
+combination of these. [UpToDate-malnutrition]_ [WHO-Malnutrition]_ 
+[emedicine-PEM]_
+
+Marasmus is characterized by a low weight-for-height and reduced mid-upper arm 
+circumference (MUAC). Other signs and symptoms include: [UpToDate-malnutrition]_
+
+  * Head that appears large relative to the body, with staring eyes
+
+  * Emaciated and weak appearance
+  
+  * Irritable and fretful affect
+
+  * Bradycardia, hypotension, and hypothermia
+
+  * Thin, dry skin
+
+  * Shrunken arms, thighs, and buttocks, with redundant skin folds caused by loss of subcutaneous fat
+
+  * Thin, sparse hair
+
+Kwashiorkor is characterized by symmetric peripheral pitting oedema that begins 
+in the lower parts of teh body and moves upwards, often affecting the presacral 
+area, genitalia, and periorbital area, with or without anasarca (severe 
+generalized oedema). There is marked muscle atrophy with normal or even 
+increased body fat. Malnutrition is considered severe if any oedema is present, 
+regardless of other anthropometric values. Other signs and symptoms include: 
+[UpToDate-malnutrition]_
+
+  * Apathetic, listless affect
+
+  * Rounded prominence of the cheeks ("moon face")
+
+  * Pursed appearance of the mouth
+
+  * Thin, dry, peeling skin with confluent areas of hyperkeratosis and hyperpigmentation
+
+  * Dry, dull, hypopigmented hair that falls out or is easily plucked
+
+  * Hepatomegaly (from fatty liver infiltrates)
+
+  * Distended abdomen with dilated intestinal loops
+
+  * Bradycardia, hypotension, and hypothermia
+
+  * Despite generalized edema, most children have loose inner inguinal skin folds
+
+Systems with mis- or impaired function due to severe acute malnutrition include: 
+cardiovascular, liver, kidney, gastrointestinal tract, immune, endocrine, 
+central nervous system, metabolism, circulation, and skin. 
+[UpToDate-malnutrition]_
+
+Due to the breadth of organ systems affected by PEM, persons suffering from PEM 
+are at increased risk for other micronutrient deficiencies, dehydration, 
+infection, and sepsis. [PCRM-PEM]_
+
+Marasmus is caused by inadequate intake of all nutrients, but particularly total 
+calories. The particular cause or nutrient deficit responsible for oedematous 
+malnutrition is still debated. [UpToDate-malnutrition]_
```

Comment:
```suggestion

```
We can delete this paragraph here since we added the details in the Marasmus/Kwashiorkor sections above.

Reply:
^ waiting for response from nicole re: the cause of oedema / protein lack

Reply:
oh man...perhaps let's have a quick chat about this to get cleared up! It seems like there are differing marasmus/kwashiorkor definitions in GBD and the literature. It would be good to note the main differences in thinking, but perhaps not too important to get into too much details cuz we are not modelling oedema.